components/clp-py-utils/clp_py_utils/clp_config.py

```diff
@@ -811,6 +811,12 @@ class CLPConfig(BaseModel):
 
     _os_release_file_path: pathlib.Path = PrivateAttr(default=OS_RELEASE_FILE_PATH)
 
+    @validator("aws_config_directory")
+    def validate_aws_config_directory(cls, value: Optional[pathlib.Path]):
```

Comment:
Actually, as we agreed a while ago, can we rename it to `expand_profile_user_home` ?